.github/workflows/build.yml

```diff
@@ -14,19 +14,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [fedora, debian]
+        name: [fedora, debian, centos]
         compiler: [gcc, clang]
         token: [softokn, softhsm]
         include:
           - name: fedora
             container: fedora:latest
           - name: debian
             container: debian:sid
+          - name: centos
+            container: quay.io/centos/centos:stream9
     container: ${{ matrix.container }}
     steps:
       - name: Install Dependencies
         run: |
-            if [ -f /etc/fedora-release ]; then
+            if [ -f /etc/redhat-release ]; then
```

Comment:
we should probably use /etc/system-release here?

Reply:
I do not have preference. If you believe the `system-release` is more appropriate, I can change it. I see its not in debian containers. But I thought the `redhat-release` was more self-documented.

Reply:
if debian does not have it no real need then.